src/main/c/ssl.c

```diff
@@ -2036,6 +2036,17 @@ TCN_IMPLEMENT_CALL(void, SSL, setState)(TCN_STDARGS,
     SSL_set_state(ssl_, state);
 }
 
+TCN_IMPLEMENT_CALL(jint, SSL, release0)(TCN_STDARGS)
```

Comment:
yeah... for example server side renegotiation will not work with boringssl. So it may be useful for a user to be able to see what can be used and what not